src/index.js

```diff
@@ -111,11 +111,11 @@ ClientApi.prototype.request = async (url, data, format) => {
   try {
     let requestConfig = defaultAxiosConfig;
     if (data) {
-      config.params = { ...config.params, ...data };
+      requestConfig.params = { ...requestConfig.params, ...data };
```

Comment:
Doesn't this cause https://github.com/zaproxy/zap-api-nodejs/pull/29#discussion_r1111799467 again?

Reply:
Oh yes, I forgot the reason behind it but linter showed me that config variable is not defined, that's why I made this change.

Reply:
It could do `let requestConfig = structuredClone(defaultAxiosConfig);`. (Node version needs to be updated to at least 17.)

Reply:
Should we do it?

Reply:
I think so.

Reply:
Node version still needs to be updated.